view/src/components/surplus.js

```diff
@@ -350,13 +408,23 @@ class Surplus extends Component {
     if (this.state.uiLoading === true) {
       return (
         <main className={classes.content}>
-          {this.state.uiLoading && <CardSkeletons classes={classes} />}
+          {this.state.uiLoading && (
```

Comment:
Are the parentheses actually needed here?

Reply:
This reformatting was done by Prettier (happy to change it back though).... let me know what you think